DOC-1860/changelog (#665)
swan-io/docs: changelog/2026/2026-07-16.mdx

---
title: 16 July 2026
type: release
---

## New features

> 💳 You can now stop a physical card from renewing automatically, without canceling it. Call the new `cancelPhysicalCardRenewal` mutation while the card is in the `ToRenew` status. The card stays active until its natural expiration date, then expires instead of being reissued.

Reviewer: ```suggestion
💳 You can now stop a physical card from renewing automatically, without canceling it. Call the new `cancelPhysicalCardRenewal` mutation while the card is in the `ToRenew` status. The card remains active until its expiry date, then expires rather than being renewed and reissued.
```

## Improvements

> 📦 You can now add several cards to Apple Pay or Google Pay in one call, with a single consent flow. The new `addDigitalCards` mutation works like `addDigitalCard`, but adds multiple cards at once.

Reviewer: ```suggestion
📦 You can now add multiple cards to Apple Pay or Google Pay with the new `addDigitalCards` mutation. It works like `addDigitalCard`, but it requires only a single consent flow for the entire batch.
```

Removing the "one call" since "single consent flow" already signals that. Using multiple cards for consistency.
